tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -635,15 +633,25 @@ def xywh2xyxy(x):
         y[:, 3] = x[:, 1] + x[:, 3] / 2
         return y
 
+    @staticmethod
+    def boxes_postprocess(boxes, img_size):
+        return boxes
+
     def process(self, raw, identifiers, frame_meta):
         result = []
         raw_outputs = self._extract_predictions(raw, frame_meta)
+        self.select_output_blob(raw_outputs)
+        self.output_name = self.output_name or self.output_blob
 
         for identifier, output, meta in zip(identifiers, raw_outputs[self.output_name], frame_meta):
-            valid_predictions = output[output[..., 4] > self.threshold]
+            mask = output[..., 4] > self.threshold
+            boxes = self.boxes_postprocess(output[:, :4], (meta['preferable_width'],
+                                                           meta['preferable_height']))[mask, :]
```

Comment:
are you sure that preferable_height and width are always in meta, they are depends from preprocessing stage. Maybe we can use more universal values?

Reply:
Done